src/main/java/cmu/pasta/mu2/diff/guidance/OptimizedMutationGuidance.java

```diff
@@ -0,0 +1,7 @@
+package cmu.pasta.mu2.diff.guidance;
```

Comment:
This doesn't need to be in the `diff` package, because it does not have anything to do with differential fuzzing.